src/test/java/uk/gov/hmcts/reform/hmc/utils/TestingUtil.java

```diff
@@ -1434,5 +1445,38 @@ public static HearingDetails hearingDetailsWithoutHearingWindow() {
     public static String formatLogMessage(String message, Object... objects) {
         return MessageFormatter.arrayFormat(message, objects).getMessage();
     }
+
+    public static HearingStatusAuditEntity hearingStatusAuditEntity() {
+        HearingStatusAuditEntity hearingStatusAuditEntity = new HearingStatusAuditEntity();
+        hearingStatusAuditEntity.setHmctsServiceId("ABA1");
+        hearingStatusAuditEntity.setHearingId("2000000000");
+        hearingStatusAuditEntity.setStatus(PutHearingStatus.HEARING_REQUESTED.name());
+        hearingStatusAuditEntity.setHearingEvent(CREATE_HEARING_REQUEST);
+        hearingStatusAuditEntity.setHttpStatus(String.valueOf(HttpStatus.SC_OK));
+        hearingStatusAuditEntity.setSource(HMC);
+        hearingStatusAuditEntity.setTarget(HMI);
+        hearingStatusAuditEntity.setRequestVersion("1");
+        return hearingStatusAuditEntity;
+    }
+
+    public static LinkedHearingStatusAuditEntity linkedHearingStatusAuditEntity() {
+        LinkedHearingStatusAuditEntity linkedHearingStatusAuditEntity = new LinkedHearingStatusAuditEntity();
+        linkedHearingStatusAuditEntity.setHmctsServiceId("Test");
+        linkedHearingStatusAuditEntity.setLinkedGroupId("2341");
+        linkedHearingStatusAuditEntity.setLinkedGroupVersion("1");
+        linkedHearingStatusAuditEntity.setLinkedHearingEventDateTime(LocalDateTime.now());
+        linkedHearingStatusAuditEntity.setLinkedHearingEvent(CREATE_HEARING_REQUEST);
+        linkedHearingStatusAuditEntity.setHttpStatus(String.valueOf(HttpStatus.SC_OK));
+        linkedHearingStatusAuditEntity.setSource(HMC);
+        linkedHearingStatusAuditEntity.setTarget(HMI);
+        JsonNode linkedGroupHearings = OBJECT_MAPPER.convertValue("2000000000", JsonNode.class);
+        linkedHearingStatusAuditEntity.setLinkedGroupHearings(linkedGroupHearings);
+        return linkedHearingStatusAuditEntity;
+    }
+
+    private static final ObjectMapper OBJECT_MAPPER = new Jackson2ObjectMapperBuilder()
```

Comment:
Not sure I like the specific reference to Jdk8. CoPilot suggests:

You can streamline the creation of an `ObjectMapper` instance with default settings, including registering modules for JDK 8 types like `Optional`, by directly instantiating `ObjectMapper` and then using its `findAndRegisterModules` method. This method automatically discovers and registers all modules on the classpath, including the `Jdk8Module`, without explicitly listing them. This approach reduces boilerplate code and ensures that your `ObjectMapper` is configured with all available modules, improving maintainability and future-proofing your serialization and deserialization logic.

Here's how you can do it:

```java
private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper().findAndRegisterModules();
```